src/components/goalDetail/GoalTodoList.tsx

```
@@ -10,11 +9,16 @@ import GoalTodoModal from './GoalTodoModal';
 import { Todo } from '@/types/todos';
 import { useDeleteGoalTodo } from '@/hooks/useGoalsTodo';
 import ConfirmModal from '../common/ConfirmModal';
+import IconButton from '../common/button/IconButton';
 
 interface Props {
   list: Todo[];
   onToggle: (id: number) => void;
```

ESLint

[error] 'id' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)